plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/sql/edit/SQLObjectEditor.java

```diff
@@ -453,8 +453,22 @@ public String getNewName() {
 
         @Override
         public DBECommand<?> merge(DBECommand<?> prevCommand, Map<Object, Object> userParams) {
+            // We need to dismiss all rename commands if there is a create command in the command queue.
+            // Otherwise we issue redundant rename commands
+            // See https://github.com/dbeaver/dbeaver/issues/11917
+            int hashCode = getObject().hashCode();
+            String createId = "create#" + hashCode;
+            Object createCmd = userParams.get(createId);
+            if (createCmd instanceof SQLObjectEditor.ObjectCreateCommand) {
+                return (SQLObjectEditor<?, ?>.ObjectCreateCommand) createCmd;
+            }
```

Comment:
Knowing that `userParams.get(createId)` will always result in object of type `SQLObjectEditor.ObjectCreateCommand` (because you're adding it to the map later), this instance check is redundant and `createCmd` can be safely casted to `SQLObjectEditor.ObjectCreateCommand`. After that, use simple null check.